transaction/creation.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2017 The Alvalor Authors
+//
+// This file is part of Alvalor.
+//
+// Alvalor is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Alvalor is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with Alvalor.  If not, see <http://www.gnu.org/licenses/>.
+
+package transaction
+
+import (
+	"encoding/binary"
+
+	"golang.org/x/crypto/blake2b"
+)
+
+// Creation creates a new account.
+type Creation struct {
+	Origin      []byte   // account issuing the creation of a new account
+	Target      []byte   // target account to be created
+	Required    uint64   // number of signatories required for a valid signature
+	Signatories [][]byte // public keys of the valid account signatories
+}
```

Comment:
No, required gives the number of signatures required from the set of signatories, so `Required <= len(Signatories)`.